### ISSUE TITLE
Bug: v1.4.03: Anti_spoof: An error occurs when set to True

### ISSUE THREAD

**Issue by yKesamaru** [yKesamaru/FACE01_SAMPLE#3]
Bug fix: v1.4.04
```python
result, score, ELE = Core_obj.return_anti_spoof(frame_datas['img'], person_data["location"])
# Bug fix
if args_dict["anti_spoof"] is True:
  if ELE is False and score > 0.8 and result == 'not_spoof':
```

**Comment by yKesamaru** [comment]
Fixed in 1.4.05

**Comment by yKesamaru** [comment]
Fixed in 1.4.05

**Comment by yKesamaru** [comment]
Fixed in 1.4.05
